Remove Java bindings since it fails to compile with Java 11.

From: Aaron Jones <aaron@vexing.codes>


---
 GNUmakefile |   11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index 482746e5..2a4a2f0d 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -364,7 +364,7 @@ include bindings/python/Makefrag
 # test scripts, etc.) in the "private" subdirectory.
 include $(wildcard private/MakefragPrivate)
 
-clean: tests-clean docs-clean tags-clean install-clean java-clean
+clean: tests-clean docs-clean tags-clean install-clean
 	rm -rf $(OBJDIR)/.deps $(OBJDIR)/*
 
 check:
@@ -420,12 +420,6 @@ INSTALL_LIBS := \
     $(OBJDIR)/libramcloud.a \
     $(NULL)
 
-# Rebuild the Java bindings
-java: $(INSTALL_LIBS)
-	bindings/java/gradlew --project-dir bindings/java
-java-clean:
-	bindings/java/gradlew --project-dir bindings/java clean
-
 # The header files below are those that must be installed in order to
 # compile RAMCloud applications. Please try to keep this list as short
 # as possible.
@@ -490,14 +484,13 @@ INSTALL_INCLUDES := \
 INSTALLED_BINS := $(patsubst $(OBJDIR)/%, $(INSTALL_DIR)/bin/%, $(INSTALL_BINS))
 INSTALLED_LIBS := $(patsubst $(OBJDIR)/%, $(INSTALL_DIR)/lib/%, $(INSTALL_LIBS))
 
-install: all java
+install: all
 	mkdir -p $(INSTALL_DIR)/bin
 	cp $(INSTALL_BINS) $(INSTALL_DIR)/bin
 	mkdir -p $(INSTALL_DIR)/include/ramcloud
 	cp $(INSTALL_INCLUDES) $(INSTALL_DIR)/include/ramcloud
 	mkdir -p $(INSTALL_DIR)/lib/ramcloud
 	cp $(INSTALL_LIBS) $(INSTALL_DIR)/lib/ramcloud
-	cp bindings/java/build/install/ramcloud/lib/* $(INSTALL_DIR)/lib/ramcloud
 
 install-clean:
 	rm -rf install
